helm/cert-operator/templates/deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       labels:
         {{- include "labels.selector" . | nindent 8 }}
       annotations:
-        releasetime: {{ $.Release.Time }}
+        releaseRevision: {{ $.Release.Revision }}
```

Comment:
```suggestion
        releaseRevision: {{ .Release.Revision }}
```

Reply:
Hmm, not according to changes @rossf7 suggested: https://github.com/giantswarm/azure-operator/commit/f2f2e06a9479135c4c0134716e947ba83dd2d384

Reply:
I talked with and `$` is a typo.

Reply:
Both work but according to the docs it should be without the `$`. 

https://helm.sh/docs/chart_template_guide/builtin_objects/

Sorry for the confusion. I'm going to fix it in my previous PRs including in our docs. 😬

Reply:
Thanks, lads. `$` removed, marking it as resolved.